test/script/evolmino.js

```diff
@@ -0,0 +1,154 @@
+ui.debug.addDebugData("evolmino", {
+	url: "5/5/i71027000222k44j23344",
+	failcheck: [
+		[
+			"arInvalid",
```

Comment:
I think this check doesn't need to be shown on the rules page. Can you add `skiprules: true`?